src/main/java/org/ihiw/management/domain/Project.java

```diff
@@ -72,7 +72,7 @@
         inverseJoinColumns = @JoinColumn(name = "ihiw_user_id", referencedColumnName = "id"))
     private Set<IhiwUser> leaders = new HashSet<>();
 
-    @OneToMany(mappedBy = "project", cascade = CascadeType.ALL, orphanRemoval = true, fetch = FetchType.LAZY)
+    @OneToMany(mappedBy = "project", cascade = CascadeType.ALL, orphanRemoval = true, fetch = FetchType.EAGER)
```

Comment:
Interesting, that this didn't already lead to propagating Uploads to the frontend...